packages/create-video/src/init.ts

```diff
@@ -61,13 +68,23 @@ xns(async () => {
 		)}. Installing dependencies...`
 	);
 	console.log('');
-	console.log('> npm install');
-	const promise = execa('npm', ['install'], {
-		cwd: outputDir,
-	});
-	promise.stderr?.pipe(process.stderr);
-	promise.stdout?.pipe(process.stdout);
-	await promise;
+	if (shouldUseYarn()) {
+		console.log('> yarn');
+		const promise = execa('yarn', [], {
+			cwd: outputDir,
+		});
+		promise.stderr?.pipe(process.stderr);
+		promise.stdout?.pipe(process.stdout);
+		await promise;
+	} else {
+		console.log('> npm install');
+		const promise = execa('npm', ['install'], {
+			cwd: outputDir,
+		});
+		promise.stderr?.pipe(process.stderr);
+		promise.stdout?.pipe(process.stdout);
+		await promise;
```

Comment:
A lot of functionality is common. I propose the following refactor:
```js
const logInitOutput = shouldUseYarn ? '> yarn' : '> npm install';
const args = shouldUseYarn ? ['yarn'] : ['npm', ['install']];

console.log(logInitOutput);
const promise = execa(...args, { cwd: outputDir });

promise.stderr?.pipe(process.stderr);
promise.stdout?.pipe(process.stdout);
await promise;
```

Variable names can be changed.